### ISSUE TITLE
Bug: Local settings don't open 3.14.7 

### ISSUE THREAD

**Issue by FadyFS** [quadproduction/OpenPype#489]
### Before posting a new ticket, have you looked through the documentation to find an answer?

Yes I have

### Have you looked through the existing tickets to find any related issues ?

Not yet

### Author of the bug

@FadyFS

### Version

3.15.11-nightly.3

### What platform you are running OpenPype on?

Linux / Centos

### Current Behavior:

the previous behavior (bug) : 
![image](https://github.com/quadproduction/OpenPype/assets/135602303/09bff9d5-3f8b-4339-a1e5-30c04ade828c)

### Expected Behavior:

![image](https://github.com/quadproduction/OpenPype/assets/135602303/c505a103-7965-4796-bcdf-73bcc48a469b)

### What type of bug is it ?

Happened only once in a particular configuration

### Which project / workfile / asset / ...

open settings with 3.14.7 

### Steps To Reproduce:

1. Run openpype on the 3.15.11-nightly.3 version 
2. Open settings in 3.14.7 version

**Comment by ClementHector** [comment]
Can you add your PR please.

**Comment by ClementHector** [comment]
You didn't follow the process indicated in fixwiki, you have extra files in your PR, now you have to rebase against ynput/develop
You should always check if you have only the changes you made in your PR:
![Image](https://github.com/quadproduction/OpenPype/assets/7068597/e19a2ea1-6c73-4359-ac7a-9030673faa51)

**Comment by FadyFS** [comment]
PR : https://github.com/ynput/OpenPype/pull/5220
